### PR TITLE
test: tests for language change

### PR DESCRIPTION
**Pull request by jzapiorkowski**
## Description

tests for language change
refactor currently existing tests

## Clickup Ticket

https://app.clickup.com/t/862hxvgma

### CODE REVIEW

File: cypress/utils/AdminPanel/LoginPageTestFunctions.ts [hunk 1]
```diff
@@ -31,4 +36,89 @@ export class LoginPageTestFunctions {
...
+        JSON.stringify({ [LoginFormFieldsNames.USERNAME]: 'admin', [LoginFormFieldsNames.PASSWORD]: 'admin' }),
+      );
+    });
+  };
+
+  emptyLoginErrorPL = () => {
+    this.getLoginInput().focus().blur();
+    this.getLoginError().should('exist');
+    this.getPasswordError().should('not.exist');
+    this.getLoginError().should('have.text', 'Login jest wymagane');
```
Comment (wgola) [inline cypress/utils/AdminPanel/LoginPageTestFunctions.ts:72]: It should be 'wymagany'

File: cypress/utils/AdminPanel/LoginPageTestFunctions.ts [hunk 2]
```diff
@@ -31,4 +36,89 @@ export class LoginPageTestFunctions {
...
+
+  private submitEmptyData = () => {
+    this.submitForm();
+    this.getLoginError().should('exist');
+    this.getPasswordError().should('exist');
+  };
+
+  submitEmptyDataPL = () => {
+    this.submitEmptyData();
+    this.getLoginError().should('have.text', 'Login jest wymagane');
```
Comment (wgola) [inline cypress/utils/AdminPanel/LoginPageTestFunctions.ts:106]: Same here

**Review by wgola — COMMENTED** [review]
Overall looks very good, but I found two spelling mistakes

File: cypress/utils/AdminPanel/LoginPageTestFunctions.ts (hunk 1 above)
Comment (jzapiorkowski) [reply cypress/utils/AdminPanel/LoginPageTestFunctions.ts]: resolved

File: cypress/utils/AdminPanel/LoginPageTestFunctions.ts (hunk 2 above)
Comment (jzapiorkowski) [reply cypress/utils/AdminPanel/LoginPageTestFunctions.ts]: resolved

**Review by wgola — APPROVED** [review]
LGTM

**Pull request merged**